### PR TITLE
Make it more clear what the difference is between URL and <url>

### PR DESCRIPTION
**Pull request by dirkschulze**
Fix for #562

### CODE REVIEW

File: master/linking.html [hunk 1]
```diff
@@ -161,14 +161,18 @@ <h3 id="URLandURI">URLs and URIs</h3>
...
-However, many <a>presentation attributes</a> allow both URLs and text strings as content.
-To disambiguate a text string from a relative URL, the <a>&lt;url&gt;</a>
-production is used for presentation attributes,
-and their corresponding CSS properties [<a href="refs.html#ref-css-values-3">css-values</a>].
-This is simply a URL delimited with a functional notation.
-</p>
+are specified using a <a>URL</a> value in an <a href="#linkRefAttrs"><span
+class="attr-name">'href'</span></a> attribute.</p>
+
+<p>This specification uses the terms <a>URL</a> and <a>&lt;url&gt;</a>.</p>
```
Comment (AmeliaBR) [inline master/linking.html]: Suggestion for the first two paragraphs, swapping the order & adding some clarification (note, I'm not sure if "apply suggestion" will work here; it only seems to work for one line at a time).

```suggestion
<p>To describe linking relationships,
this specification uses two different data types in attribute and property values:
<a>URL</a> and <a>&lt;url&gt;</a>.
The linking guidelines in this chapter apply to URLs specified with either syntax.
</p>

<p>The <a>URL</a> data type is a simple URL string.
In SVG, most structural relationships between two elements
are specified using a <a>URL</a> value in an <a href="#linkRefAttrs"><span
class="attr-name">'href'</span></a> attribute.
</p>
```

(and then continue with the paragraph about the `<url>` data type)

Comment (svgeesus) [reply master/linking.html]: @dirkschulze please accept (beta) the suggested change from @AmeliaBR and reply about the target of "must not"?

File: master/linking.html [hunk 2]
```diff
@@ -161,14 +161,18 @@ <h3 id="URLandURI">URLs and URIs</h3>
...
+
+<p><a>&lt;url&gt;</a> is different from <a>URL</a> and represents a CSS <code>url()</code> function value.
+(See CSS Values and Units for further details [<a href="refs.html#ref-css-values-3">css-values</a>]). <a>&lt;url&gt;</a>s
+may be used for <a>presentation attributes</a> and their corresponding CSS properties
+[<a href="refs.html#ref-css-values-3">css-values</a>].</p>
+
+<p><a>URL</a> must not be a valid value for <a>presentation attributes</a> for structural
+relationships purposes. No non-<a>presentation attribute</a> allows <a>&lt;url&gt;</a> as value.</p>
```
[the remark this answers is not in the document]
Comment (svgeesus) [reply master/linking.html]: Maybe "is not" ?IOW not a conformance clause, just an observation.

File: master/linking.html [hunk 3]
```diff
@@ -161,14 +161,18 @@ <h3 id="URLandURI">URLs and URIs</h3>
...
+class="attr-name">'href'</span></a> attribute.</p>
+
+<p>This specification uses the terms <a>URL</a> and <a>&lt;url&gt;</a>.</p>
+
+<p><a>&lt;url&gt;</a> is different from <a>URL</a> and represents a CSS <code>url()</code> function value.
+(See CSS Values and Units for further details [<a href="refs.html#ref-css-values-3">css-values</a>]). <a>&lt;url&gt;</a>s
+may be used for <a>presentation attributes</a> and their corresponding CSS properties
+[<a href="refs.html#ref-css-values-3">css-values</a>].</p>
+
+<p><a>URL</a> must not be a valid value for <a>presentation attributes</a> for structural
```
Comment (svgeesus) [inline master/linking.html]:
```suggestion
<p><a>URL</a> is not a valid value for <a>presentation attributes</a> for structural
```

**Pull request merged**